README.md

```diff
@@ -638,6 +640,53 @@ handle dependencies and CLDR loading manually yourself.
   [Read more...](doc/error/e-unsupported.md)
 
 
+## Contribution
```

Comment:
Actually, "Contribution" is more parallel than "Contributing" given the subsequent section "Development".